search-engines.lisp

```diff
@@ -1074,6 +1074,134 @@ REQUEST-ARGS is a list of args to pass to request function."
      :completion-function (make-pkgs-completion)
      :documentation "UNIX/GNU/Linux package search."))
 
+(define-search-engine peertube
+    (:shortcut "pt"
```

Comment:
A small note: `pt` is too short, it can clash with the engines others defined. Try to use long name of the search engine instead.